docs/source/en/model_doc/align.md

```diff
@@ -13,70 +13,182 @@ specific language governing permissions and limitations under the License.
+</hfoption>
+<hfoption id="AutoModel">
 
-```python
+```py
+from transformers import AlignProcessor, AlignModel
+from PIL import Image
 import requests
 import torch
-from PIL import Image
+
+# device
+device = "cuda" if torch.cuda.is_available() else "cpu"
+
+# 1. Load processor and model
+processor = AlignProcessor.from_pretrained("kakaobrain/align-base")
+model     = AlignModel.from_pretrained("kakaobrain/align-base").to(device)
+model.eval()
+
+# 2. Fetch and open image
+url      = "https://huggingface.co/roschmid/dog-races/resolve/main/images/Golden_Retriever.jpg"
+response = requests.get(url, stream=True)
+image    = Image.open(response.raw).convert("RGB")
+
+# 3. Prepare inputs
+#   a) image embeddings
+image_inputs = processor(images=image, return_tensors="pt").to(device)
+with torch.no_grad():
+    image_embeds = model.get_image_features(**image_inputs)
+
+#   b) text embeddings
+candidate_labels = ["a photo of a dog", "a photo of a cat", "a photo of a person"]
+text_inputs      = processor(text=candidate_labels, padding=True, return_tensors="pt").to(device)
+with torch.no_grad():
+    text_embeds = model.get_text_features(**text_inputs)
+
+# 4. Normalize embeddings
+image_embeds = image_embeds / image_embeds.norm(p=2, dim=-1, keepdim=True)
+text_embeds  = text_embeds  / text_embeds.norm(p=2, dim=-1, keepdim=True)
+
+# 5. Compute logits and probabilities
+logits = (image_embeds @ text_embeds.T) * 100.0
+probs  = logits.softmax(dim=-1).cpu().squeeze()
+
+# 6. Display results
+for label, score in zip(candidate_labels, probs):
+    print(f"{label:20s} → {score.item():.4f}")
+```
+
+</hfoption>
+<hfoption id="transformers-cli">
+
+```py
+# this command downloads the kakaobrain/align-base for offline use
+transformers-cli download kakaobrain/align-base
+```
+
+</hfoption>
+</hfoptions>
+
+### Quantization
+
+Quantizing `align-base` to 8-bit or 4-bit does not reduce memory usage—in fact, it can increase it. This is because the model is relatively small, and components like the vision encoder aren’t optimized for quantized execution. Quantization benefits are typically seen in larger models (e.g., LLMs) where memory and compute reductions are more significant. Use full precision unless you're experimenting or working under extreme constraints.
+
+---
+
+### Attention Mask Visualization
+
+ALIGN is a **dual‑encoder** (separate image+text) model with **no** autoregressive decoding head, so it does **not** support AttentionMaskVisualizer.  
+That utility only works on models with a `.generate()` step that emits token‑level attentions (e.g. GPT‑style or encoder‑decoder models).
+
+## Notes
+
+- ALIGN uses a dual-encoder architecture with EfficientNet for images and BERT for text processing
+- The model is designed for zero-shot image-text retrieval tasks
+- Pre-trained on 1.8B image-text pairs from the web
```

Comment:
Already mentioned the architecture and pretraining earlier so no need to include here

```suggestion
- ALIGN projects the text and visual features into latent space and the dot product between the projected image and text features is used as the similarity score. The example below demonstrates how to calculate the image-text similarity score with [`AlignProcessor`] and [`AlignModel`].
```